test/widgets/message_list_test.dart

```diff
@@ -981,4 +981,70 @@ void main() {
       });
     });
   });
+
+  group('TypingStatus', () {
+    final users = [eg.selfUser, eg.otherUser, eg.thirdUser, eg.fourthUser];
+    final finder = find.descendant(
+      of: find.byType(TypingStatusWidget),
+      matching: find.byType(Text)
+    );
+
+    checkTyping(WidgetTester tester, TypingEvent event, {required String expected}) async {
+      await store.handleEvent(event);
+      await tester.pump();
+      check(finder.evaluate()).single.has((x) => x.widget, 'widget').isA<Text>()
+        .data.equals(expected);
```

Comment:
`tester.widget` can simplify this:
```suggestion
      check(tester.widget<Text>(finder)).data.equals(expected);
```